Src/BootCamp.Chapter/BinaryConverter.cs

```diff
@@ -0,0 +1,19 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Text;
+
+namespace BootCamp.Chapter
+{
+    public static class BinaryConverter
+    {
+        public static long ToInteger(string binary)
+        {
+            return Conversions.ConvertBinaryToDecimal();
+        }
+
+        public static string ToBinary(long number)
+        {
+            return Conversions.ConvertDecimalToBinary(); ;
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Remove this empty statement.](https://app.codacy.com/manual/Almantask/CSharp-From-Zero-To-Hero/pullRequest?prid=5002934)